functional/FunctionalTest.java

```diff
@@ -3558,9 +3559,8 @@ public static void main(String[] args) throws Exception {
     } else {
       kmsKeyName = System.getenv("MINIO_JAVA_TEST_KMS_KEY_NAME");
```

Comment:
Can we remove this environment variable?

Reply:
I don't why it is useful though @kannappanr